packages/jest-runtime/src/index.ts

```diff
@@ -833,6 +844,31 @@ class Runtime {
     this._moduleMocker.clearAllMocks();
   }
 
+  // fileUrl.protocol cannot be set to a non-standard protocol, so we use string manipulation
+  private _createOutsideJestVmPath(path: string) {
```

Comment:
these don't have to be on `this`, do they?

Reply:
Nope, can move outside the class if that's your preference?

Reply:
the class is like 1000 lines 😅